TestAssets/NonRestoredTestProjects/DotnetAddP2PProjects/ValidRef/ValidRef.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net451;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net451;netcoreapp1.0;netstandard1.4</TargetFrameworks>
```

Comment:
this is just for testing conditions and this is to make sure that this project will be compatible with all other tfms used in this dir